File: app/src/main/java/com/originb/inkwisenote/modules/handwrittennotes/PageBackgroundType.java
package com.originb.inkwisenote.modules.handwrittennotes;

public enum PageBackgroundType {
    BASIC_RULED_PAGE_TEMPLATE("BASIC_RULED_PAGE_TEMPLATE");

    private final String pageTemplateName;

    PageBackgroundType(String pageTemplateName) {
        this.pageTemplateName = pageTemplateName;
    }

    public String getPageTemplateName() {
        return pageTemplateName;
    }
}

File: app/src/main/java/com/originb/inkwisenote2/modules/handwrittennotes/PageBackgroundType.kt
package com.originb.inkwisenote2.modules.handwrittennotes

enum class PageBackgroundType(pageTemplateName: String) {
    BASIC_RULED_PAGE_TEMPLATE("BASIC_RULED_PAGE_TEMPLATE");

    val pageTemplateName: String?

    init {
        this.pageTemplateName = pageTemplateName
    }
}
